introduction.md

```diff
@@ -6,41 +6,170 @@ layout: documentation
+**When home automation just seems to work, it is always the result of hard work.** 
+Home automation is fascinating and requires a considerable investment of your time. 
+Here are some key considerations especially for new users. To be successful, you will need to:
+
+* **Start slowly and one step at a time.**
+* Be prepared to learn
+* Remain flexible in how you want to achieve your goal
+* Celebrate all the small successes
+
+Remember, openHAB is just a computer program. 
+The computer will only do what *you* tell him to do.
+openHAB can provide many default solutions that are easy to setup. 
+On the flip side, the more you insist that everything should look and work exactly as you want it, the more work you will have to do. 
+openHAB is fully customizable, but doing so will require substantial efforts from you.
+
+After you have read the documentation for openHAB, you will have:
+
+* Identified on which computer you will run openHAB
+* Learned how to install openHAB, as well as all other software that is needed to run openHAB (e.g., JAVA)
+* Learned how your smart devices communicate with openHAB; how to make openHAB give commands to your smart devices; and how you can interact with openHAB
+
+Keep your focus. 
+For almost everything, there is more than one way in openHAB to achieve a goal or perform a task. 
+Initially, this can be frustrating and confusing, but it also gives great flexibility and does not mandate the use of certain devices or tools.
+
+You may need to learn more about computers that you currently do.
+But this may not be a problem as you are reading the documentation of software that will enable you to do home automation yourself.
+
+Lastly, be prepared to start a new hobby: home automation. 
+Basic functions can be achieved in openHAB rather quickly, e.g., switch lights on at a certain time. 
+Others will require much more efforts and thoughts, e.g., how do I determine that someone is home, including guests, but not counting pets? 
+The openHAB forum is a great place to learn and discuss. 
+But always remember: no one but you knows what you need and what you have as well as yourself. 
+Give plenty of background information if you have a question. 
+And please be courteous and respectful, this is an open-source community and everybody on the forum works with openHAB in their spare time.
+
+## A quick overview
+
+openHAB communicates electronically with smart and not-so-smart devices, performs user-defined actions and provides web-pages with user-defined information as well as user-defined tools to interact with all devices. 
+To achieve this, openHAB segments and compartmentalizes certain functions and operations. 
+The following table gives a high level description of the most important concepts as well as a link to more information:
+
+| Concepts | Meaning | More Information |
+| --- | --- | --- |
+| Bindings | are the openHAB component that provides the interface to interact electronically with devices | _see below_ |
```

Comment:
Was the limitation to "Bindings" made an purpose?

I am asking because:
In fact Bindings are only a sub category off all add-ons.
Add-ons is the navigation title in paper ui. 
Add-ons is the used term for the website category.

I know that this aims to be a simplified overview, but i would think about using addons, to help people find their way in the website navigation/paper ui (and possibly other places as well).

Reply:
Very self-centered approach: I remember my confusion about the various add-ons while I was still struggling to understand the basic concepts. Not sure how to include transforms, persistence, integrations, voice and actions without writing a whole para. I wanted to leave it just as a straight line: device connects via binding to thing, which connects to item, which I can see on sitemap. Adding the other add-ons means adding a whole bunch of side chains....happy to get pointers if I am overthinking it. Otherwise, it would be my preference to leave as is

Reply:
Fine for me, just wanted to talk about it.
We can change it anyway later, when someone has a better approach. :)